### PR TITLE
Update things to work with ghc 8.8.3

### PR DESCRIPTION
**Pull request by adithyaov**

### CODE REVIEW

**Comment by adithyaov** [comment]
TODO:
- [x] Add a CI for linux
- [x] Fixup commits

File: bench-show.cabal [hunk 1]
```diff
@@ -60,10 +60,7 @@ description:
 category:            Performance, Benchmarking
 homepage:            https://github.com/composewell/bench-show
 license-file:        LICENSE
-tested-with:           GHC==8.6.5
-                     , GHC==8.4.4
-                     , GHC==8.2.2
-                     , GHC==7.10.3
+tested-with:         GHC==8.8.3
```
Comment (harendra-kumar) [inline bench-show.cabal:63]: Since nothing has changed in the source, it should still be compiling with all other compilers. It won't hurt to add 8.6.5/8.4.4 to the build matrix and still have those here.

File: app/Main.hs [hunk 2]
```diff
@@ -18,7 +18,6 @@
 --
 module Main where
 
-import Data.Semigroup ((<>))
```
Comment (harendra-kumar) [inline app/Main.hs:21]: Is this change required or just a warning fix?

**Review by harendra-kumar — COMMENTED** [review]
Looks like there are no changes that mandate a new release. I am able to compile the hackage version with ghc 8.8. So we can commit these changes for CIs, but do not need to upload a release.

File: app/Main.hs (hunk 2 above)
Comment (adithyaov) [reply app/Main.hs:21]: It's a warning fix

**Review by harendra-kumar — APPROVED** [review]
